tests/test_preprints.py

```diff
@@ -462,13 +467,114 @@ def test_format_preprint(self):
 
         assert nodes == {}
 
+    def test_format_thesis(self):
```

Comment:
This is a lot just for the one change. If we're super-concerned about the change, it might be worth modifying the original test to run through all of the potential categories. Alternately, for this test we might just verify that the 'thesis' comes through and don't worry so much about all the rest of the things. Having two almost identical complicated tests is generally not ideal, mostly because if we change something (like, say, getting all the metadata from the preprint object instead of half from there and half from node) we don't want to have to change both of those tests.